google/cloud/bigquery/job.py

```diff
@@ -1990,8 +1990,11 @@ class ExtractJob(_AsyncJob):
     Args:
         job_id (str): the job's ID.
 
-        source (google.cloud.bigquery.table.TableReference):
-            Table into which data is to be loaded.
+        source (Union[ \
+            google.cloud.bigquery.table.TableReference, \
+            google.cloud.bigquery.model.ModelReference \
+        ]):
+            Table or Model from which data is to be loaded.
```

Comment:
Maybe s/loaded/extracted/ in the comment.

Reply:
@shollyman  We don't have any model lifecycle test in the integration tests.